README.md

```diff
@@ -15,6 +15,19 @@ composer create-project drupal/starshot-project --repository='{"type":"vcs","url
 ```
 This one command will install Starshot and open it in a web browser for you to play with. You'll get all the modules and themes listed below, pre-configured.
 
+### DDEV installation
+If you don't have PHP and Composer installed on your host machine, you can try in DDEV:
+```
+mkdir starshot && cd starshot
+ddev config --project-type=drupal --docroot=web
+ddev start
+ddev composer create drupal/starshot-project --repository='{"type":"vcs","url":"https://github.com/phenaproxima/starshot-prototype"}' --stability=dev
+```
+After the download completes, you probably get an error since `drupal quick-start` doesn't work in DDEV. Stop the server with Ctrl-C, and create login link:
```

Comment:
@phenaproxima
This line basically creates a new usage pathway - looks like users cannot use a single-line command to just try it. 

What are the other commands apart from `drupal quick-start` that would not work in the chain of composer `scripts` in `composer.json`?

I would highly recommend adding a CI pipeline with DDEV installed that would be executing these steps before adding such instructions to make sure that these instructions aer bullet-proof and work 100%.

Reply:
I like the idea of a CI pipeline confirming it works with DDEV, but I don't see it getting prioritized. @TravisCarden is the CI expert on this project, but if you want to set up a PR that adds a workflow to install Starshot on DDEV, I'd happily merge it.

And IMHO, it's okay if the DDEV usage pathway is more than a single line. Down the road, we could probably offer some sort of DDEV-specific script to make it work with one line. Or, folks can just copy and paste these few simple commands. It's not a big deal.

Reply:
I agree so far, that it would be ideal if the process could either use quickstart or DDEV, depending on the environment  ... But for now, the minimal steps to get it running in DDEV seems worthwhile.